.azure-pipelines/azure-pipelines-image-template.yml

```diff
@@ -31,8 +31,8 @@ jobs:
           if [ -n "$(CACHE_MODE)" ] && echo $(PLATFORM_AZP) | grep -E -q "^(vs|broadcom|mellanox|marvell-armhf)$"; then
             CACHE_OPTIONS="SONIC_DPKG_CACHE_METHOD=$(CACHE_MODE) SONIC_DPKG_CACHE_SOURCE=/nfs/dpkg_cache/$(PLATFORM_AZP)"
             BUILD_OPTIONS="$(BUILD_OPTIONS) $CACHE_OPTIONS"
-            echo "##vso[task.setvariable variable=BUILD_OPTIONS]$BUILD_OPTIONS"
           fi
+          echo "##vso[task.setvariable variable=BUILD_OPTIONS]$BUILD_OPTIONS"
```

Comment:
>echo [](http://example.com/codeflow?start=10&length=4)

Why echo statement could fix anything? Or do you want to change PR title?

Reply:
Thanks, updated the title, and added more detail info in the description.